pkg/virtctl/network/dynamicifaces.go

```diff
@@ -36,8 +36,8 @@ import (
 const (
 	HotplugCmdName = "addinterface"
 
-	ifaceNameArg   = "iface-name"
-	networkNameArg = "network-name"
+	ifaceNameArg                       = "iface-name"
```

Comment:
Why didn't you rename `iface-name` to `name`?

Reply:
Added a commit to change it as well.